next/.gitignore

```diff
@@ -37,4 +37,5 @@ yarn-error.log*
 # Sentry
 .sentryclirc
 
-
+# local env files
+.env
```

Comment:
Hmm we usually do not gitignote `.env`, only `.env*.local` which is already gitignored few lines above.